### PR TITLE
Add Basic Layouts docs and TS examples

### PR DESCRIPTION
**Pull request by yuriy-fix**

### CODE REVIEW

**Comment by sosa-vaadin** [comment]
image
In small screens, the content overflows the container for this example. Maybe wrapping should be enabled here.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        <vaadin-radio-button value="auto" checked>Auto (default)</vaadin-radio-button>
+        <vaadin-radio-button value="stretch">Stretch</vaadin-radio-button>
+        <vaadin-radio-button value="flex-start">Start</vaadin-radio-button>
+        <vaadin-radio-button value="center">Center</vaadin-radio-button>
+        <vaadin-radio-button value="flex-end">End</vaadin-radio-button>
+        <vaadin-radio-button value="baseline">Baseline</vaadin-radio-button>
+      </vaadin-radio-group>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```
Comment (sosa-vaadin) [inline frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts:72]: Same issue about mixing TS and HTML content.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-vertical-alignment.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      >
+        <vaadin-radio-button value="stretch" checked>Stretch (default)</vaadin-radio-button>
+        <vaadin-radio-button value="flex-start">Start</vaadin-radio-button>
+        <vaadin-radio-button value="center">Center</vaadin-radio-button>
+        <vaadin-radio-button value="flex-end">End</vaadin-radio-button>
+        <vaadin-radio-button value="baseline">Baseline</vaadin-radio-button>
+      </vaadin-radio-group>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```
Comment (sosa-vaadin) [inline frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-vertical-alignment.ts:53]: Same issue about mixing TS and HTML content.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
```
Comment (sosa-vaadin) [inline frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout.ts:1]:
```suggestion
import 'Frontend/demo/init'; // hidden-source-line
```
Change this comment wherever it appears in every example.
https://github.com/vaadin/docs-app/pull/256

**Review by sosa-vaadin — CHANGES_REQUESTED** [review]
Replace `hidden-full-source-line` comments with `hidden-source-line` and fix snippet tags placements.

File: articles/ds/components/basic-layouts/index.asciidoc [hunk 4]
```diff
@@ -6,15 +6,369 @@ section-nav: incomplete
...
+//----
+//include::{root}/src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayout.java[render,tags=snippet,indent=0,group=Java]
+//----
+--
+
+Like Vertical Layout, Horizontal Layout enables both vertical and horizontal alignment of components.
+
+=== Vertical Alignment
+
+You can position components at the top, middle, or bottom. Items can alternatively be made to stretch vertically or positioned along the layout’s text baseline.
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply articles/ds/components/basic-layouts/index.asciidoc]: Done.

File: frontend/demo/component/basiclayouts/basic-layouts-expanding-items.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      <vaadin-radio-group
+        label="Item 1: sizing"
+        @value-changed="${(e: RadioGroupValueChangedEvent) => (this.size = e.detail.value)}"
+      >
+        <vaadin-radio-button value="0" checked>Default size</vaadin-radio-button>
+        <vaadin-radio-button value="1">Expand</vaadin-radio-button>
+      </vaadin-radio-group>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-expanding-items.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-horizontal-alignment.ts [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <vaadin-radio-button value="flex-start" checked>Start (default)</vaadin-radio-button>
+        <vaadin-radio-button value="center">Center</vaadin-radio-button>
+        <vaadin-radio-button value="flex-end">End</vaadin-radio-button>
+        <vaadin-radio-button value="space-between">Between</vaadin-radio-button>
+        <vaadin-radio-button value="space-around">Around</vaadin-radio-button>
+        <vaadin-radio-button value="space-evenly">Evenly</vaadin-radio-button>
+      </vaadin-radio-group>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-horizontal-alignment.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts (hunk 1 above)
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-vertical-alignment.ts (hunk 2 above)
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-vertical-alignment.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts (hunk 1 above)
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-horizontal-alignment.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-horizontal-alignment.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-expanding-items.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-expanding-items.ts]: Updated.

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout.ts (hunk 3 above)
Comment (yuriy-fix) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout.ts]: Updated.

**Review by sosa-vaadin — APPROVED** [review]

**Pull request merged**